### PR TITLE
fix: Support Excalidraw inside scrollable container

### LINKED ISSUE #3011: Support Excalidraw inside scrollable container
Tasklist to support Excalidraw inside a scrollable container

- [x] Remove position fixed completely https://github.com/excalidraw/excalidraw/issues/2737.
- [x] Calculate offsets when the page is scrolled so all interactions work properly.

This will partly fix https://github.com/excalidraw/excalidraw/issues/2991

### PR DESCRIPTION
**Pull request by ad1992**
fixes #3011 &  #2737 

For host check it out [here](https://k1xx5.csb.app)

TODOS
- [x] Removed position fixed from everywhere except for context menu since it's rendered inside excalidraw class so top needs to set from the viewport. This will not create any issues in the host as well so should be fine.
- [x] Account for offsets when the page is scrolled 
- [x] Testing
- [x] update changelog

### CODE REVIEW

**Comment by dwelle** [comment]
- [ ] fix hitbox testing and everything around scene manipulation

**Review by dwelle — APPROVED** [review]

**Review by lipis — APPROVED** [review]

File: src/components/App.tsx [hunk 1]
```diff
@@ -999,6 +1000,10 @@ class App extends React.Component<ExcalidrawProps, AppState> {
     }
   }
 
+  private onScroll = debounce(() => {
+    this.setState({ ...this.getCanvasOffsets() });
+  }, 500);
```
Comment (lipis) [inline src/components/App.tsx:1005]: can we add this magic number to: `constants.ts`

File: src/components/LayerUI.tsx [hunk 2]
```diff
@@ -442,7 +442,13 @@ const LayerUI = ({
         "transition-left": zenModeEnabled,
       })}
     >
-      <Island className={CLASSES.SHAPE_ACTIONS_MENU} padding={2}>
+      <Island
+        className={CLASSES.SHAPE_ACTIONS_MENU}
+        padding={2}
+        style={{
+          maxHeight: `${appState.height - 200}px`,
```
Comment (lipis) [inline src/components/LayerUI.tsx:449]: Another magic number.. what's this `200`?

File: src/components/App.tsx (hunk 1 above)
Comment (ad1992) [reply src/components/App.tsx]: 👍 probably we should have `time_constants` file for all time related constants

Comment (lipis) [reply src/components/App.tsx]: Let's keep them all there now.. and we'll split them if gets out of hand.

**Pull request merged**